cookbook/readme.md

```diff
@@ -0,0 +1,155 @@
+# DISCLAIMER !
+**Examples provided in this cookbook are still _under construction_, so if you see some sections/folders that are empty or incomplete it is because we are still working on it.
+You can find here the list of what is [Completed](#Completed), [In progress](#In progress) or [Not started](#Not started).**
+
+## Completed
+* [direct-database-actions/basic-crud-actions](plugins/direct-database-actions/basic-crud-actions)
+
+## In progress
+* [direct-database-actions](plugins/direct-database-actions)
+
+## Not started
+* [common-camel-patterns](plugins/common-camel-patterns)
+* [notifications-and-emails](plugins/notifications-and-emails)
+* [slimsgate-xml-advanced-patterns](plugins/slimsgate-xml-advanced-patterns)
+* [ui-only-actions-as-code](plugins/ui-only-actions-as-code)
+
+# How to use this cookbook
+
+**This section assumes you have already followed the instructions in the [top-level readme](../README.md) to setup your gradle.properties
+and run the `./gradlew idea` or `./gradlew eclipse` command successfully**
+
+**For extremely basic examples that can be used as jumping off points for new plugins, please see [templates](../templates)
+and the [top-level readme](../README.md) in this repo**
+
+This cookbook is intended to provide you with a series of example plugins and code samples for developing SLIMS plugins 
+utilizing the slimsgate api. It is not intended to replace the slims development manual, [Apache's camel documentation](https://camel.apache.org/components/4.0.x), 
```

Comment:
Here I think we should not put 4.0.x version of camel doc -> since we are still in 3.x 
https://camel.apache.org/components/3.14.x/index.html